frontend/src/app/store.ts

```diff
@@ -1,6 +1,5 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
```

Comment:
Frage aus Interesse: Redux Toolkit, klingt vielversprechend auf der Webseite. Was kann man damit vereinfachen?

Reply:
redux toolkit vereinfacht das redux state management.
Ich habe ehrlich gesagt kaum mit redux ohne das toolkit gearbeitet. Daher kann ich dir nicht genau ausfuehren, was es einfacher macht.
Ein Punkt, den ich sehr schaetze ist z.B., dass man mittels der ThunkApi auf states aus anderen Bereichen des Stores zugreifen kann oder actions in einer action dispatchen